src/data/faq.json

```diff
@@ -2702,7 +2703,7 @@
                 {
                     "term": "Effective validity",
                     "anchor": "effective_validity",
-                    "description": "We refer to the duration of vaccination protection as 'effective validity'. The effective validity of the vaccination thus counts from the last vaccination date. The effective validity of the vaccination protection may vary from country to country. In the case of a <a href='#glossary_recovery_certificate'>recovery certificate</a>, the effective validity is set at 'date of the person's first positive <a href='#glossary_pcr_test'>PCR test</a> result' plus 180 days, according to the EU eHealth Network. See also <a href='#eu_dcc_validity' target='_blank' >\"How long is my certificate valid?\". </a>"

```

Comment:
This is using `markdown `syntax in a `json `file. That doesn't work.

`[recovery certificate](https://www.coronawarn.app/en/faq/#glossary_recovery_certificate)`

needs to be changed to use HTML.